cmds/core/chroot/chroot.go

```diff
@@ -196,6 +192,14 @@ func main() {
 	}
 
 	if err = cmd.Run(); err != nil {
+		return err
+	}
+	return nil
+}
+
+func main() {
+	flag.Parse()
+	if err := chroot(flag.NArg(), flag.Args()...); err != nil {
```

Comment:
I don't see the reason for passing flag.Narg() given that we can get Nargs from len(flag.Args())

Reply:
I was confused about this. Fixed it now. Thank you for catching it.